Cookbook/Technical-Documents/Current-guide.md

```diff
@@ -0,0 +1,29 @@
+# `Current` Guide - how to use it in Babylon iOS Project
+
+### Where is that? 
+- `Current` is the global instance of the `World` type which is located in the `BabylonDependencies.framework`. 
+Its purpose is to provide convenient access to the instances that should be shared between different parts of the app (cross-cutting concerns). For more information please read the [How Control the World](/Cookbook/Proposals/ControlTheWorld.md) document.
+
+## `Do`s and `Don't`s
+
+#### Do
+- access `Current` **only** from topmost `AppDelegate` until `Builder`s layer
```

Comment:
```suggestion
- You should only access `Current` in builders, not in ViewModels or Renderers. The AppDelegate is included
```

Reply:
- lowercase, and not in first person
- the **not in** is part of the `Don't` section